File: example/javalib/config/2-testing/integration/src/qux/QuxIntegrationTests.java
package qux;

import org.junit.jupiter.api.Test;

public class QuxIntegrationTests {

  @Test
  public void helloworld() {
    String result = Qux.hello();
    QuxTests.assertHello(result);
    QuxTests.assertWorld(result);
  }
}

File: example/kotlinlib/testing/3-integration-suite/qux/integration/src/qux/QuxIntegrationTests.kt
package qux

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.shouldBe

class QuxIntegrationTests :
    FunSpec({

        test("helloworld") {
            val result = Qux.hello()
            result shouldBe "Hello World"
        }
    })
